### ISSUE TITLE
Add support for pinning sidebar

### ISSUE THREAD

**Issue by ridhwaans** [a85/POSTMan-Chrome-Extension#7383]
**Is your feature request related to a problem? Please describe.**
Not related

**Describe the solution you'd like**
The postman electron app on mac (and windows) doesnt have a visible pin to keep the sidebar open when the window isnt maximized. This results in toggling the sidebar from the bottom left corner
**Describe alternatives you've considered**
n/a
**Additional context**

**Comment by ilons** [comment]
I got the same issue, this was previously working just fine (side panel expanded by default), and I didn't event know you could close it.
My use-case is that I have postman open in a 1280px wide window, which is not fullscreen. 
Having to expand the side panel each time I need to change to another request makes it really painful to work with, and will push me away real fast as a user.

**Edit:** After updating to 7.7.3 this is no longer the case it seems.

**Comment by mackaaij** [comment]
Super annoying indeed. About two weeks ago, Postman decided to hide the sidebar at launch and I use it every time to navigate to the calls I saved. Would be nice to be able to pin the sidebar.

**Comment by mmxcrono** [comment]
Yes, please give me this option instead of forcing me to have it auto-hide.

**Comment by Pijuli** [comment]
Same here. Why is it hidden on launch on FHD screen? :/

**Comment by nagu** [comment]
Autohide should be disabled by default and need a config to set it.

**Comment by Dangerunicorn** [comment]
I am currently experiencing this issue with a user I support. Her sidebar stays hidden even when I toggle the icon. I can't find a way to get it to appear.

**Comment by mackaaij** [comment]
Somehow, it’s working for me as desired for a few weeks now. I didn’t do anything - although I might have updated Postman.

**Comment by Dangerunicorn** [comment]
We were able to remedy my user's issue by using a larger monitor. Unfortunately she cannot access the left side menu when using her laptop alone.

**Comment by neoramos** [comment]
Please add setting to maintain sidebar open.  As others have suggested, perhaps a setting "auto-hide" one can toggle to turn on and off feature would be great.  I use a vertical monitor and don't like that the sidebar is closed when I move postman to that screen.
Thank you.
